linode/manager
# change: [M3-7396] - Update `axios` to `1.6.1`

## Description 📝

- Update `axios` to latest 🎉
  - For CVE-2023-45857 (https://github.com/linode/manager/security/dependabot/88)
- Updates `tsup` to latest so that we are on the latest esbuild
  - We needed to do this because the new axios code uses JS features that the older esbuild version didn't support with our current tsup configuration https://github.com/evanw/esbuild/pull/3194 

## How to test 🧪
 
- Test Cloud Manager in general
- Verify file uploads for Object Storage and Images still work
- Verify all CI and e2e pass

## As an Author I have considered 🤔
